conversiontracerbullet/src/main/java/org/stanford/MarcToXML.java

```diff
@@ -37,12 +37,12 @@
  */
 class MarcToXML {
 
-    private static Connection authDB = null;
+    public static Connection authDB = null;
```

Comment:
have you made these public so the tests can see them?  If so, does "package"  (no qualifier) or "protected work?